t/020-basic.t

```diff
@@ -37,7 +37,7 @@ if library-exists('portaudio', v2) {
         ok $stream.active, "and it is active now";
         is $stream.info.sample-rate, 44100e0, "got some info";
         nok $stream.stopped, "and it is obviously not stopped anymore";
-        ok $stream.write-available, "and write-available gives us something";
+        cmp-ok $stream.write-available, '<=', 0, "and write-available gives us something non-negative";
```

Comment:
i don't have a lot of experience submitting pull requests yet. i didn't mean for the callback example to be part of this PR. it's a work in progress. on my system it segfaults after a few seconds, but it does make sound before it crashes.

regarding the bad arrow: oops. sorry and thanks for even looking at this stuff :)